app/views/step_by_step_pages/publish_or_delete.html.erb

```diff
@@ -23,43 +23,57 @@
+              <p class="govuk-body">
                 Scheduled to be published at <%= @step_by_step_page.scheduled_at %>
-              </div>
+              </p>
               <%= form_tag step_by_step_page_unschedule_path(@step_by_step_page), method: :unschedule do %>
-                <button class="btn btn-danger">Unschedule publishing</button>
+                <%= render "govuk_publishing_components/components/button", {
+                  text: "Unschedule publishing",
+                  destructive: true
+                } %>
               <% end %>
             <% else %>
-              <%= link_to 'Schedule publish', step_by_step_page_schedule_path(@step_by_step_page), class: "btn btn-info" %>
+              <%= render "govuk_publishing_components/components/button", {
+                text: "Schedule publish",
+                href: step_by_step_page_schedule_path(@step_by_step_page),
+                secondary: true
+              } %>
             <% end %>
           <% end %>
 
           <% if @step_by_step_page.unpublished_changes? %>
-            <p>Or</p>
-            <%= button_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: "btn btn-danger" %>
+            <p class="govuk-body govuk-!-margin-top-5">Or</p>
+            <%= button_to 'Discard changes', { action: "revert", step_by_step_page_id: @step_by_step_page.id }, data: { confirm: 'This will delete your draft. Do you want to discard your changes?' }, class: "gem-c-button govuk-button gem-c-button--secondary" %>
           <% end %>
         </div>
       </div>
     <% end %>
-    <div class="panel panel-danger delete-or-unpublish-panel">
+    <div class="govuk-panel panel-danger delete-or-unpublish-panel">
       <div class="panel-body">
         <% if @step_by_step_page.has_been_published? %>
-          <%= link_to 'Unpublish step by step', step_by_step_page_unpublish_path(@step_by_step_page), class: "btn btn-danger" %>
+          <%= render "govuk_publishing_components/components/button", {
+            text: "Unpublish step by step",
+            href: step_by_step_page_unpublish_path(@step_by_step_page),
+            destructive: true
+          } %>
         <% else %>
-          <p class="text-danger"><strong>Warning</strong></p>
-          <p>Deleted step by steps cannot be recovered.</p>
-          <%= link_to 'Delete step by step', @step_by_step_page, method: :delete, data: { confirm: 'Delete this step by step page?' }, class: "btn btn-danger" %>
+          <p class="text-danger govuk-body"><strong>Warning</strong></p>
+          <p class="govuk-body">Deleted step by steps cannot be recovered.</p>
+          <%= button_to 'Delete step by step', @step_by_step_page, method: :delete, data: { confirm: 'Delete this step by step page?' }, class: "gem-c-button govuk-button govuk-button--warning" %>
```

Comment:
I think this too should be a "destructive" button as well.

Reply:
`govuk-button--warning` class adds the "destructive" mode